src/components/molecules/Comment/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { CommentData } from "@/types/commentData";
+import CircularProfileImage from "@/components/atoms/CircularProfileImage";
+import ChildComment from "../ChildComment";
+
+export interface CommentWithChild extends CommentData {
+  childComments: CommentData[];
+}
+
+interface Props {
+  comment: CommentWithChild;
+}
+
+function Comment({ comment }: Props) {
+  return (
+    <>
+      <div className="flex items-center gap-3">
+        <CircularProfileImage src="/images/default_profile_image.png" styleType="lg" />
+        <div className="flex-1">
+          <div className="flex items-center justify-between">
+            <span className="text-[#2a5885]">{comment.userName}</span>
+            <div className="text-neutral-400">
+              <span className="text-sm">답글 달기</span>
+            </div>
+          </div>
+          <pre className="whitespace-pre-wrap break-all">{comment.content}</pre>
+        </div>
+      </div>
+      <hr className="mt-2" />
+      {comment.childComments &&
+        comment.childComments.map((childComment) => <ChildComment childComment={childComment} key={childComment.id} />)}
```

Comment:
optional chaining 써서 축약할 수 있을 것 같습니다!
```suggestion
      {comment.childComments?.map((childComment) => <ChildComment childComment={childComment} key={childComment.id} />)}
```

Reply:
완료